lib/facter/pe_status_check.rb

```diff
@@ -203,6 +203,20 @@
     { S0022: validity }
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+
+    maximum = PEStatusCheck.max_connections.to_i
+    current = PEStatusCheck.cur_connections.to_i
+    percent_used = (current / maximum) * 100
+    { S0029: percent_used >= 90 }
+    rescue ZeroDivisionError
```

Comment:
this is flagging as syntaxual incorrect in rubocop - but its not

its valid syntax for ruby 2.5 and above, our template is using ruby 2.4 for some reason in rubo cop, i will correct this